socorro/external/es/base.py

```diff
@@ -4,6 +4,88 @@
 
 import datetime
 
+from configman import Namespace, class_converter
+
+from socorro.app.generic_app import App
+
+
+def string_to_list(input_str):
+    return [x.strip() for x in input_str.split(',') if x.strip()]
+
+
+class ElasticsearchConfig(App):
+
+    required_config = Namespace()
+    required_config.namespace('elasticsearch')
+    required_config.elasticsearch.add_option(
+        'elasticsearch_class',
+        doc='a class that implements the ES connection object',
+        default='socorro.external.es.connection_context.ConnectionContext',
+        from_string_converter=class_converter
+    )
```

Comment:
Fixed now. 

Note that the `ConnectionContext` class defines [elasticsearch_timeout](https://github.com/peterbe/socorro/blob/bug-1188087-supersearchfields-directly-via-ssocorro/socorro/external/es/connection_context.py#L52) but it doesn't define `elasticsearch_timeout_extended` which I've left in here. It seems to be used only by `socorro.external.es.query.Query`. Kinda strange that it's separated like that.